src/styles/index.css

```diff
@@ -1,18 +1,72 @@
 /**
- * Homepage styles
  *
- * Warning: This styling is ultimately applied to the global scope!
+ * Homepage (index.js) Styling
+ *
+ * Warning: This styling is ultimately applied to the global scope! Use specific
+ * classes to make sure it only applies to the homepage.
+ *
  */
 
-
 .main-header { margin: 3rem 0 5rem 0; }
 
+/**
+ * The art row is a collage of some of our illustrations - we use the buildings
+ * as the main basis for the collage and then absolutely position our clean
+ * vehicles and appliances around them
+ */
+.art-row {
+    position: relative;
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-top: 3rem;
+    margin-bottom: 6rem;
+}
+
+.art-row * {
+    filter:  drop-shadow(-0.25rem 0.25rem 0rem var(--light-shadow-color));
+}
+
+.art-row .appliance-sheet, .art-row .car-sheet {
```

Comment:
Man I really need to get Sass setup so I can do nesting and not have to right this cludgy CSS 😆